.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
```

Comment:
What is this?

Reply:
Should not be there: removed the directory and added to the gitignore